### ISSUE TITLE
Add support for reading auth-basic info from a file

### ISSUE THREAD

**Issue by dongursli** [xWyatt/check_rest_api#8]
Now this is done when command is run and therefore password is shown in `ps`

```
 -b, --auth-basic=<username>:<password>
    Uses HTTP Basic authentication
```

**Comment by xWyatt** [comment]
Oops, that was a bug. I have that fixed and re-released with version 1.0.1.

Download that and give it a try. If it still isn't working let me know.

Thanks!
W

**Comment by xWyatt** [comment]
I've added this feature to the current release (1.1.0). 

Thanks for the suggestion!
W
